jobbergate/cli.py

```diff
@@ -152,7 +152,7 @@ def ask_questions(fields, answerfile, use_defaults=False):
     # Check if questions has already been answered
     for question in flatten(questions):
         if question.name in answerfile:
-            retval.update({question.name: answerfile[question.name]})
+            pass
         elif use_defaults and question.default is not None:
```

Comment:
Can't the `if question.name in answerfile` be removed and the `elif` be changed to `if`?

Reply:
Not outright, pre-supplied answers take precedence.
If it is already answered, both using defaults and actually asking the question should not happen.

We could do something like :
```
if not already answered and [-f flag]:
    use default
elif not already answered:
    ask the question
```

But I think the current one is slightly clearer in what's going on.